FreeRTOS/iot-reference-arm-corstone3xx
# Add Corstone-310 target support

Description
-----------
This PR adds new target Corstone-310 support to FRI. With this PR, Corstone-310 target becomes the default target for building and running the FRI. In addition, [eco-system FVPs](https://developer.arm.com/downloads/-/arm-ecosystem-fvps) are used by default while running the FRI.

Test Steps
-----------
Tested blinky, AWS IoT core end to end connectivity and OTA job.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
